test/job_queue_test.exs

```diff
@@ -37,6 +35,9 @@ defmodule EctoJobScheduler.JobQueueTest do
 
         job = Repo.one(unquote(job_queue))
 
+        Mox.expect(ReporterMock, :stop_transaction, fn _ -> :ok end)
+        Mox.expect(ReporterMock, :fail, fn _ -> :ok end)
```

Comment:
here should not fail because it is a success case, I need to investigate further

@rai200890 @richardherald @yashin5